.github/workflows/delivery/docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine:3.10
+
+ARG VERSION
```

Comment:
I used `VERSION` over `PACK_VERSION`, because empirically it seemed that `outputs` didn't get set for things with a `_`.

Reply:
Oh hmm... I didn't observe this before. That said, the use of `VERSION` here seems pretty clear.